src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -115,6 +115,13 @@ public void clickCollapseExpandButton() {
         click(collapseExpandButton);
     }
 
+    public void expandPanels() {
+        if (isElementPresent("collapse-panels-button")) {
+            clickCollapseExpandButton();
+            waitForPanelsToExpand();
+        }
+    }
```

Comment:
In `InstructorFeedbackResultsPageUiTest`  there is similar method `clickCollapseExpandButtonAndWaitForPanelsToExpand` . We can now remove that method and to use this method ? 

There are other methods like `clickAjaxLoadedPanelAndWaitForExpansion` and  `clickCollapseExpandButtonAndWaitForPanelsToCollapse` in that class. Shouldn't they be in `InstructorFeedbackResultsPage` instead of `InstructorFeedbackResultsPageUiTest` ?